sidebars.js

```diff
@@ -49,6 +49,11 @@ module.exports = {
       collapsed: true,
       label: 'Go SDK',
       items: [
+        {
+          type: 'link',
+          label: 'API Reference',
```

Comment:
is there some way we can mark these as external links? would be a nice to have the svg icon in here, or failing which, an emoji.

Reply:
I will take note of this and see about doing some customizations there -

Reply:
can use 🔗 emoji if we can't use the external link svg

Reply:
Cool. Now looks like this:
![Screen Shot 2020-11-10 at 8 49 37 AM](https://user-images.githubusercontent.com/34380806/98697069-bd3a0d00-2331-11eb-98e1-c3567ec1b8f8.png)